### PR TITLE
Work for ts 66

### PR DESCRIPTION
**Pull request by reshmabidikar**
- Added configuration code
- Added healthcheck code
- Added basic testing code (more tests need to be added)
- Added ci workflow

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "work-for-release-0.23.x"
```
Comment (pierre) [inline .github/dependabot.yml]: `main` :-)

**Review by pierre — APPROVED** [review]
Could you just look into the failing test? Are we missing configuration properties maybe?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      test-profile-matrix: '[ "travis", "integration-mysql", "integration-postgresql" ]'
+    secrets:
+      extra-env: '{ "GOCARDLESS_ACCESS_TOKEN": "${{ secrets.GOCARDLESS_ACCESS_TOKEN }}", "GOCARDLESS_ENVIRONMENT": "${{ secrets.GOCARDLESS_ENVIRONMENT }}"}'
+
+  integration_test:
+    uses: killbill/gh-actions-shared/.github/workflows/integration_test.yml@main
+    with:
+      plugin_name: 'gocardless'
+      integration_tests_goal: 'test:plugins:gocardless'
+      integration_tests_ref: 'refs/heads/integration-tests-for-gocardless'
```
Comment (pierre) [inline .github/workflows/ci.yml]: PR was merged.

File: .github/dependabot.yml (hunk 1 above)
Comment (reshmabidikar) [reply .github/dependabot.yml]: Fixed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (reshmabidikar) [reply .github/workflows/ci.yml]: Fixed

**Comment by reshmabidikar** [comment]
> Could you just look into the failing test? Are we missing configuration properties maybe?

The tests fail since the Gocardless credentials are not set up as Github secrets. Could you please set the following:

- `GOCARDLESS_ACCESS_TOKEN` - Gocardless access token
- `GOCARDLESS_ENVIRONMENT` - Environment to use. Should be set to `sandbox`

**Comment by pierre** [comment]
Done! Merging to check CI...

**Pull request merged**